includes/database/engine/class-table.php

```diff
@@ -337,13 +350,13 @@ public function exists() {
 	}
 
 	/**
-	 * Check if table already exists
+	 * Create the table.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 *
-	 * @return bool|void
+	 * @return bool
```

Comment:
```suggestion
	 * @return bool|void
```

due to:

```php
// Bail if no database interface is available
if ( empty( $db ) ) {
	return;
}
```

Reply:
Now that you point it out, I don't like that they return null/void. They could return `false` which is both more predictable and results in simpler documentation.

Reply:
Yes that makes more sense.